.github/workflows/master.yml

```diff
@@ -0,0 +1,156 @@
+name: master
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+# We're doing pytest with `xvfb-run` for CI since without this some of the tests would fail for GUI library because there is no display server.
```

Comment:
Can you move this comments right above `PYTHON_EXECUTABLE`?  Also, please keep these comments with the 100 char length to make it easier to line.